README.md

```diff
@@ -119,14 +121,51 @@ $ docker run -ti --rm \
     skycoin/skycoin
 ```
 
-With this image, the skycoin daemon will run as root by default inside the
-container. When you mount a volume in /data, the container will detect for the
-folder's owner UID and will create a skycoin user with the same UID.
+This image has a `skycoin` user for the skycoin daemon to run, with UID and GID 10000.
+When you mount the volumes, the container will change their owner, so you
+must be aware that if you are mounting an existing host folder any content you
+have there will be own by 10000.
+
+The container will run with some default options, but you can change them
+by just appending flags at the end of the `docker run` command. The following
+example will show you the available options.
+
+```sh
+docker run --rm skycoin/skycoin -help
+```
 
 Access the dashboard: [http://localhost:6420](http://localhost:6420).
 
 Access the API: [http://localhost:6420/version](http://localhost:6420/version).
 
+### Building your own images
+
+There are some folders in the `docker/images/` folder. You can build different
+images with the Dockerfiles they contain for different purposes.
+
+#### Mainnet
+
+When you build this Dockerfile, docker will download the source code from
+github on the specified tag and make an image out of it. The stable image
+should come from this one. The `--build-arg` is optional as there is a default
+in the Dockerfile.
+
+```sh
+$ cd docker/images/mainnet
+$ docker build -t skycoin:stable --build-arg=SKYCOIN_VERSION=0.22.0
+```
+
+#### Snapshot
+
+This Dockerfile it's used to build the current state of the source you are
```

Comment:
_the source you are working on_ => _your local working copy or develop branch_ .